packages/react-accordion/etc/react-accordion.api.md

```diff
@@ -9,64 +9,150 @@ import type { ComponentProps } from '@fluentui/react-utilities';
 import type { ComponentState } from '@fluentui/react-utilities';
 import type { Context } from '@fluentui/react-context-selector';
 import type { IntrinsicShorthandProps } from '@fluentui/react-utilities';
-import type { ObjectShorthandProps } from '@fluentui/react-utilities';
+import { ObjectShorthandProps } from '@fluentui/react-utilities';
 import * as React_2 from 'react';
+import { ShorthandProps } from '@fluentui/react-utilities';
 import { ShorthandRenderFunction } from '@fluentui/react-utilities';
 
 // @public
-export const Accordion: React_2.ForwardRefExoticComponent<AccordionProps & React_2.RefAttributes<HTMLDivElement>>;
+export const Accordion: React_2.ForwardRefExoticComponent<Pick<{
```

Comment:
We talked about this a bit, but I'll ask here. 
Do we know a way to avoid, mitigate, or reduce this type explosion? 
It being future work is OK.

Reply:
Explicitly declaring the types fixes it. `export const Accordion: SomeExplicitType = ...`